frontend/src/core/comments/components/Comment.css

```diff
@@ -19,7 +19,6 @@
     display: flex;
     font-size: 11px;
     padding: 6px;
-    white-space: pre-wrap;
```

Comment:
I don't understand your comment - why do we need to remove this declaration?

Reply:
The comments are saved as strings in the original implementation of comments. However, with this implementation they are saved as html, so each line is a separate `<p>` . The `pre-wrap`, of course, handles each differently.  So, consideration needs to be made on how to handle previous comments that have line breaks in them.

<img width="365" alt="without-prewrap" src="https://user-images.githubusercontent.com/19564687/78740756-78b64100-790c-11ea-841b-56d56048161b.png">


<img width="365" alt="with-prewrap" src="https://user-images.githubusercontent.com/19564687/78740771-8370d600-790c-11ea-9c09-35c59115f990.png">

Reply:
OK, I see why removing `pre-wrap makes sense. We can convert existing comments to the new format with a data migration. I think that answer your point 3.